components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/RestApiIntegrationTest.java

```diff
@@ -337,6 +370,58 @@ public void testQuery() throws Exception {
         assertNotNull(queryRecords);
     }
 
+    @Test
+    public void testParentRelationshipQuery() throws Exception {
+        final QueryRecordsContact queryRecords = template().requestBody("direct:parentRelationshipQuery", null,
```

Comment:
Are we missing bits to create Contacts with Accounts here, i.e. can this test be run without other tests?